### ISSUE TITLE
Download not working?

### ISSUE THREAD

**Issue by TBRC-pmench** [buda-base/public-digital-library#214]
This may be a problem with my local setup, but I wasn't able to download W1NLM436. Clicking on dowload PDF/zip gave no result.
![ezgif com-optimize (3)](https://user-images.githubusercontent.com/48484924/90133792-14820e80-dd3e-11ea-80ba-b2c51277c5e2.gif)

**Comment by eroux** [comment]
ah, interesting, thanks! @MarcAgate can you take a look at the iiif server logs?

**Comment by TBRC-pmench** [comment]
I should add that I also tried downloading Vol. 1 of W3CN21368, PDF, to no avail (just keeps spinning and spinning and spinning; I'll keep the tab open and if successful update the comment).

**Comment by eroux** [comment]
apparently the IIIF server works fine for the PDF download... @berger-n maybe it's a bug in the client? can you take a look?

**Comment by eroux** [comment]
@TBRC-pmench @MarcAgate and @berger-n can't reproduce the problem... can you still experience it? if so maybe there's an error message in the console?

**Comment by TBRC-pmench** [comment]
Let me try again and I'll look at the console. I'll disable my extensions as well.

**Comment by TBRC-pmench** [comment]
No luck. I can try on a different computer as well later today and give an update.

**Comment by eroux** [comment]
part of what's happening is that it's not clear that users need to reclick on PDF once the spinner is done spinning... perhaps the download should start automatically?

**Comment by TBRC-pmench** [comment]
Yes, I was able to download with no issue once I knew to re-click the PDF link after the file finished loading.

**Comment by eroux** [comment]
So I think we have a few options...
- indicate more clearly that users should click on the button... perhaps by making it a real, bigger button with 3 states:
   * Generate PDF
   * Generating PDF...
   * Download PDF
with a small blinking when the "Download PDF" state is reached... I think that can be a first step
- make the download start automatically once the PDF is generated

@berger-n I let you pick the one you prefer, both work for me

**Comment by xristy** [comment]
FWIW, option 1 is essentially what happens currently on tbrc.org.

**Comment by eroux** [comment]
yes, I think it's good. We went a step too far in the minimalization

**Comment by berger-n** [comment]
@eroux first option was quite straightforward to implement
second one is tricky as a bit of googling has just reminded me... 

![image](https://user-images.githubusercontent.com/33383857/90156220-7d907300-dd8c-11ea-8d35-181500fdba67.png)

I encountered a few unexpected errors 502 taking that screenshot though...
I think there should be a fourth state like "server error - retry"
I'm on it

**Comment by eroux** [comment]
ok thanks! @MarcAgate what does the log of the iiif server say about these 502 errors?

**Comment by berger-n** [comment]
it happens when requesting 5-6 volumes at once:
![image](https://user-images.githubusercontent.com/33383857/90160286-dd3d4d00-dd91-11ea-9b9e-27da9b913ff3.png)

**Comment by eroux** [comment]
thanks! moving to https://github.com/buda-base/buda-iiif-server/issues/58
